fact_extractor/plugins/unpacking/tpl/test/test_tp_wr702n.py

```diff
@@ -10,54 +12,62 @@
 
 
 class TestTpWr702n(TestUnpackerBase):
-    def setUp(self):
-        super().setUp()
+    def setup_method(self):
+        super().setup_method()
 
         self.test_firmware = os.path.join(TEST_DATA_DIR, 'wr.fw')
         self.firmware_container = TPWR702N(self.test_firmware)
 
-    def tearDown(self):
+    def teardown_method(self):
```

Comment:
These functions could theoretically be replaced by fixtures, but I'd be to lazy to do it.

Reply:
I know but it was out of scope for this PR. I would consider it "future work". The PR only aims at "fixing" the warnings.